argo-cd-apps/base/member/infra-deployments/knative-eventing/knative-eventing.yaml

```diff
@@ -13,9 +13,19 @@ spec:
               values:
                 sourceRoot: components/knative-eventing
                 environment: staging
-                clusterDir: ""
+                clusterDir: base
           - list:
-              elements: []
+              elements:
+                - nameNormalized: kflux-ocp-p01
+                  values.clusterDir: kflux-ocp-p01
+                - nameNormalized: stone-stage-p01
+                  values.clusterDir: stone-stage-p01
+                - nameNormalized: stone-prod-p01
+                  values.clusterDir: stone-prod-p01
+                - nameNormalized: stone-prod-p02
+                  values.clusterDir: stone-prod-p02
+                - nameNormalized: kflux-prd-rh02
+                  values.clusterDir: kflux-prd-rh02
```

Comment:
can you add kflux-prd-rh03, this is a new cluster we are building now, it is almost ready

Reply:
Added, let me know if its OK.